### PR TITLE
Adding helper function to scrape webpage and save books by chapter.

### PR DESCRIPTION
**Pull request by zmacks**
Adding the --dl_chapters argument will download all books in chapters by scraping the webpage of the original book for PDF links. Please, I don't have a ton of experience working in groups, so would love constructive feedback. Thanks!

### CODE REVIEW

**Comment by zmacks** [comment]
Forgot to mention, only supports PDF

File: requirements.txt [hunk 1]
```diff
@@ -10,6 +12,9 @@ python-dateutil==2.8.1
 pytz==2019.3
 requests==2.23.0
 six==1.14.0
+soupsieve==2.0
 tqdm==4.45.0
-urllib3>=1.25.8
+urllib3==1.25.8
 xlrd==1.2.0
+urllib3>=1.25.8
```
Comment (chaosAD) [inline requirements.txt]: The error is telling you that you have duplicate dependency. See the error details [here](https://github.com/alexgand/springer_free_books/pull/94/checks?check_run_id=657950986).

Comment (zmacks) [reply requirements.txt]: Thanks for pointing me in the right direction, @chaosAD. Checks all pass now :)

**Review by alexgand — APPROVED** [review]
Merged, thanks! Can you modify the README.MD file to show the new usability?

**Comment by alexgand** [comment]
Hi @zmacks, I really liked this option. But it would be nice if the chapters were saved in order of appearence (say, putting 01, 02, ..., in front of the title of the chapter in the filename), so it would appear in correct order in the downloaded folder. Can you make this happen?
Also, please update the README.MD to show the new option.
Thanks.

**Comment by zmacks** [comment]
Thanks @alexgand ! I submitted a new [pull request](https://github.com/alexgand/springer_free_books/pull/100) to address those concerns : )

**Pull request merged**